.travis.yml

```diff
@@ -17,5 +17,5 @@ install:
   - travis_retry python setup.py dev
 script: py.test --runslow
 after_success:
-  - coveralls
+  - coveralls --config_file=.coveragerc_nogpu
```

Comment:
Ah, hadn't seen that we can just alter the `py.test` run for Travis instead. Will update.